### PR TITLE
few more spelling changes

### PR DESCRIPTION
**Pull request by jennyf19**
@jmprieur when do we want to remove the migration aid?

### CODE REVIEW

File: src/client/Microsoft.Identity.Client/MigrationAid/MigrationAid.cs [hunk 1]
```diff
@@ -390,7 +390,7 @@ public partial interface IPublicClientApplication
     {
 #if WINDOWS_APP
         /// <summary>
-        /// Flag to enable authentication with the user currently logeed-in in Windows.
+        /// Flag to enable authentication with the user currently logged-in on Windows.
```
Comment (henrik-me) [inline src/client/Microsoft.Identity.Client/MigrationAid/MigrationAid.cs:393]: >logged-in [](start = 66, length = 9)

signed-in ?

File: src/client/Microsoft.Identity.Client/MigrationAid/MigrationAid.cs [hunk 2]
```diff
@@ -808,7 +808,7 @@ public partial class PublicClientApplication
     {
 #if WINDOWS_APP
         /// <summary>
-        /// Flag to enable authentication with the user currently logged-in in Windows.
+        /// Flag to enable authentication with the user currently logged-in on Windows.
```
Comment (henrik-me) [inline src/client/Microsoft.Identity.Client/MigrationAid/MigrationAid.cs:811]: >logged-in [](start = 66, length = 9)

I suppose this is used in many places, wondering if we should call it signed-in everywhere

**Review by henrik-me — APPROVED** [review]
:shipit:

File: src/client/Microsoft.Identity.Client/MigrationAid/MigrationAid.cs (hunk 2 above)
Comment (jennyf19) [reply src/client/Microsoft.Identity.Client/MigrationAid/MigrationAid.cs:811]: idk

Comment (jennyf19) [reply src/client/Microsoft.Identity.Client/MigrationAid/MigrationAid.cs]: done.

**Pull request merged**